### DISCUSSION TITLE
mspider/user-guide/cluster-management/remove-clus

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#3729] [Announcements]
# mspider/user-guide/cluster-management/remove-clus

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/user-guide/cluster-management/remove-clus.html

**Comment by nicole-lihui** [comment]
移除检测可以增加图例
